apps/web/src/app/[locale]/(dashboard)/[wsId]/tasks/boards/enhanced-boards-view.tsx

```
@@ -0,0 +1,2328 @@
+  useEffect(() => {
+    setCurrentPage(1);
+  }, [filters.selectedBoard, filters.statusFilter, searchQuery]);
+
+  const getStatusColor = (status: string) => {
+    switch (status) {
+      case 'done':
+      case 'closed':
+        return 'bg-green-500';
+      case 'active':
+        return 'bg-blue-500';
+      case 'not_started':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)